### PR TITLE
Add emoji to links on README

### PR DESCRIPTION
**Pull request by DNin01**
- Adds icons next to some of the links.
- Makes the most important links bold.
- Changes "Useful links for contributors" to "Shortcuts for contributors".

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -21,26 +21,26 @@ Scratch Addons is not affiliated, associated, or in any way officially connected
 
 ### 🔧 Browser extension
 
-**GitHub repository:** [ScratchAddons/ScratchAddons](https://github.com/ScratchAddons/ScratchAddons) | [README](https://github.com/ScratchAddons/ScratchAddons#readme) | [Issues](https://github.com/ScratchAddons/ScratchAddons/issues) | [Pull requests](https://github.com/ScratchAddons/ScratchAddons/pulls) | [Discussions](https://github.com/ScratchAddons/ScratchAddons/discussions)<!-- 2 spaces -->  
+**GitHub repository: [📁 ScratchAddons/ScratchAddons](https://github.com/ScratchAddons/ScratchAddons) | [🪧 README](https://github.com/ScratchAddons/ScratchAddons#readme) | [📝 Issues](https://github.com/ScratchAddons/ScratchAddons/issues) | [💫 Pull requests](https://github.com/ScratchAddons/ScratchAddons/pulls) | [👥 Discussions](https://github.com/ScratchAddons/ScratchAddons/discussions)**<!-- 2 spaces -->
```
Comment (WorldLanguages) [inline profile/README.md:24]: The README emoji is too new. I can't see it on Windows 10

Comment (DNin01) [reply profile/README.md:24]: It's a signpost. iOS and Windows 11 have it, but I guess Windows 10 is still missing that emoji.

Comment (WorldLanguages) [reply profile/README.md:24]: Can you change it to another emoji?

Comment (DNin01) [reply profile/README.md:24]: Alright.

**Comment by WorldLanguages** [comment]
What do you think? @Hans5958 😄

**Review by Hans5958 — CHANGES_REQUESTED** [review]
No need for links to be bold

**Comment by DNin01** [comment]
> No need for links to be bold

I was trying to make the most useful links for users more prominent, but I suppose most of the people coming here are probably more interested in the source code.

File: profile/README.md [hunk 2]
```diff
@@ -37,9 +37,9 @@ Shortcuts for contributors:
 
 ### 🌐 Website
 
-**GitHub repository: [📁 ScratchAddons/website-v2](https://github.com/ScratchAddons/website-v2) | [ℹ️ README](https://github.com/ScratchAddons/website-v2#readme) | [📝 Issues](https://github.com/ScratchAddons/website-v2/issues) | [💫 Pull requests](https://github.com/ScratchAddons/website-v2/pulls) | [👥 Discussions](https://github.com/ScratchAddons/ScratchAddons/discussions) | [📖 Wiki](https://github.com/ScratchAddons/website-v2/wiki)**
+**GitHub repository:** [📁 ScratchAddons/website-v2](https://github.com/ScratchAddons/website-v2) | [ℹ️ README](https://github.com/ScratchAddons/website-v2#readme) | [📝 Issues](https://github.com/ScratchAddons/website-v2/issues) | [💫 Pull requests](https://github.com/ScratchAddons/website-v2/pulls) | [👥 Discussions](https://github.com/ScratchAddons/ScratchAddons/discussions) | [📖 Wiki](https://github.com/ScratchAddons/website-v2/wiki)
 
-**Website available at: [🌐 ScratchAddons.com](https://scratchaddons.com), [ScratchAddons.org](https://scratchaddons.org)**
+**Website available at:** [🌐 ScratchAddons.com](https://scratchaddons.com), [ScratchAddons.org](https://scratchaddons.org)
```
Comment (WorldLanguages) [inline profile/README.md:42]:
```suggestion
**Website available at:** [ScratchAddons.com](https://scratchaddons.com), [ScratchAddons.org](https://scratchaddons.org)
```

**Review by WorldLanguages — APPROVED** [review]

**Comment by DNin01** [comment]
@WorldLanguages @Hans5958 Still waiting...

**Pull request merged**